lib/codemod/README.md

```diff
@@ -237,3 +239,39 @@ import { Meta, Story } from '@storybook/addon-docs/blocks';
   <Button label="Story 2" onClick={action('click')} />
 </Story>
 ```
+
+### convert-mdx-to-module
+
+This converts all your MDX stories into module format.
+
+```sh
+./node_modules/.bin/jscodeshift -t ./node_modules/@storybook/codemod/dist/transforms/convert-to-module-format.js . --ignore-pattern "node_modules|dist" --extensions=mdx
+```
+
+For example:
+
+```js
+import React from 'react';
+import Button from './Button';
+import { Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title='Button' />
+
```

Comment:
Any markdown here just gets completely dropped?

Reply:
For now, yes

Reply:
Hmm.. do you think this is useful if we do that?

Reply:
Absolutely. We're going from a format that supports mixtures of longform documentation and code to one that does not. This completely solves the tedious task of translation in an automated fashion, and for now users will have to figure out themselves how they want to encode that documentation.

If we ever have a recommended way to encode longform docs in module format, we can try to encode that in the codemod.

Reply:
I thought you could put markdown in the module format somehow?

Reply:
I don't have a prescription for that. Some users will put it in docgen comments, others in strings, others in `.md` files that get loaded in. When somebody has a good solution they want to push, they can upgrade this codemod accordingly.

I personally think people should be using MDX for longform docs and not module format, and the purpose of this PR is to help people who try it and want to go back for whatever reason.